### ISSUE TITLE
Toggle section Header does not highlight when used in a stack

### ISSUE THREAD

**Issue by geethakavalgar** [cerner/terra-core#3630]
# Bug Report

## Description
When the toggle section header is created in stack like multiple section header, the highlighting does not work. The last and first highlights fine, but the middle one does not highlight properly.

## Steps to Reproduce

1. Add the section header in a stack.
2. Collapse one by one to see the highlighting issue.

## Additional Context / Screenshots
![Pane highlighting behavior](https://user-images.githubusercontent.com/17171030/174148665-686b7e96-7752-49c2-a6d7-593947ba3ff7.gif)

## Expected Behavior
The section header should highlight when it is collapsed.

## Environment

* https://engineering.cerner.com/terra-ui/components/cerner-terra-core-docs/toggle-section-header/about and 2.58.2
* Browser Name and Version:* Node/npm Version: [Node 10/npm 6.x]
* Webpack Version:
* Operating System and version (desktop or mobile): desktop

## @ Mentions
@BenBoersma

**Comment by chrismichalewicz** [comment]
UXPLATFORM-6766 has been logged to track this internally. At this time we do not have a timeframe for when we'd be able to make this update.
